editoast/src/models/paced_train.rs

```diff
@@ -18,6 +18,7 @@ use crate::models::prelude::*;
 use crate::models::train_schedule::TrainSchedule;
 
 #[derive(Debug, Clone, Model)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
Note that if `PartialEq` is derived only for tests, then the same should be done for all the other objects on which deriving `PartialEq` (and `Eq`) has been done.

Reply:
We can’t, the others objects are part of a different crate, editoast_schema. It’s the drawback of splitting into different crates.